### PR TITLE
MsGraph get attachment fix

### PR DESCRIPTION
**Pull request by AradCarmi**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-4641

## Description
Fixed an issue when parsing an email with an attachment via the `email communication` and `MsGraph mail` packs, the attachment wasn't handled correctly.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/EmailCommunication/Scripts/PreprocessEmail/PreprocessEmail.py [hunk 1]
```diff
@@ -183,14 +177,14 @@ def check_incident_status(incident_details, email_related_incident):
...
-def get_attachments_using_instance(email_related_incident, labels):
+def get_attachments_using_instance(email_related_incident, labels, email_to):
     """Use the instance from which the email was received to fetch the attachments.
         Only supported with: EWS V2, Gmail
 
     Args:
         email_related_incident (str): ID of the incident to attach the files to.
         labels (Dict): Incident's labels to fetch the relevant data from.
-
+        attachment_id
```
Comment (Shellyber) [inline Packs/EmailCommunication/Scripts/PreprocessEmail/PreprocessEmail.py:187]: What's this?

**Review by Shellyber — COMMENTED** [review]
Nice job!
Let's sync with Meital regarding the BC if its the best practice.

File: Packs/EmailCommunication/Scripts/PreprocessEmail/PreprocessEmail.py (hunk 1 above)
Comment (AradCarmi) [reply Packs/EmailCommunication/Scripts/PreprocessEmail/PreprocessEmail.py:187]: Leftover good catch!

Comment (AradCarmi) [inline Packs/EmailCommunication/Scripts/PreprocessEmail/PreprocessEmail.py:187]:
```suggestion
```

**Review by Shellyber — COMMENTED** [review]
Let's just add a test

File: Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphMail/MicrosoftGraphMail_test.py [hunk 2]
```diff
@@ -652,6 +652,35 @@ def test_get_attachment(client):
         assert output.get(output_prefix).get('Subject') == 'Test it'
 
 
+@pytest.mark.parametrize('client', [oproxy_client(), self_deployed_client()])
+def test_get_attachments_without_attachment_id(mocker, client):
+    """
+    Given:
+        - raw response returned from get_attachment_command
```
Comment (AradCarmi) [inline Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphMail/MicrosoftGraphMail_test.py:659]:
```suggestion
        - Raw response returned from get_attachment_command
```

**Review by ShahafBenYakir — APPROVED** [review]

**Pull request merged**